server/server.go

```diff
@@ -37,6 +38,15 @@ var gracefulServer *graceful.Server
 func Start() {
 	LogInfo("Push proxy server is initializing...")
 
+	if CfgPP.ProxyServer != "" {
```

Comment:
Does this mean that Android already obeys the `HTTP_PROXY`/`HTTPS_PROXY` environment variables?